dashboard/config/courses/csa-pilot.course

```diff
@@ -17,7 +17,17 @@
     "family_name": "csa"
   },
   "resources": [
-
+    {
+      "name": "All Standards",
```

Comment:
Is adding this for CSA intentional?

Reply:
oh shoot, no it was part of testing. I'll remove